style.css

```diff
@@ -0,0 +1,118 @@
+/* For mobile phones */
+@media only screen and (max-width: 600px) {
+  * {
+    box-sizing: border-box;
+    margin: 0;
+  }
+
+  body {
+    background-color: #1c1a19;
+    height: 50vh;
+    background-image: url("./Images/Illustration.svg");
```

Comment:
- [ ] Kindly fix the background image as it iscurrently not displaying.

Your design
![image](https://user-images.githubusercontent.com/81932478/185350079-1965f9ca-4e4a-4ff8-a2bb-8ac28891b28d.png)

Provided design
![image](https://user-images.githubusercontent.com/81932478/185350286-9a06f3bb-38e5-49cd-ae9e-f25b1ff0e1d8.png)